internal/pkg/build/remotebuilder/remotebuilder.go

```diff
@@ -107,7 +106,9 @@ func (rb *RemoteBuilder) Build(ctx context.Context) (err error) {
 	if rb.IsDetached {
 		fmt.Printf("Build submitted! Once it is complete, the image can be retrieved by running:\n")
 		fmt.Printf("\tsingularity pull --library %s library://%s\n\n", bi.LibraryURL, libraryRefRaw)
```

Comment:
```suggestion
		fmt.Printf("\tsingularity pull library://%s\n\n", bi.LibraryURL, libraryRefRaw)
```

Shouldn't the use of the `--library` argument be deprecated since the advent of `singularity remote`?

Reply:
You can still specify a custom `--library` flag, and this far down in the code we can't tell whether the library URI came from the remote config, or a custom flag.

We can get rid of it, which is neater and correct for 99.9% of cases where it should be the remote defined library.... or we can leave it in case.